### PR TITLE
Created Alert and Avtar with different styles

### PR DESCRIPTION
**Pull request by sanu29**
Alert and Avtar with different styles added.

image
image
image
image

### CODE REVIEW

File: component.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
+/* @import url("./components/card.css"); */
```
Comment (zoef-yt) [inline component.css:2]: remove commented code.

File: components/alert.css [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+.alert {
+    display: flex;
+    color: #fff;
+    padding: 1rem 1.5rem 1rem 1.5rem;
+    margin: 10px;
+    border-radius: 5px;
+    font-size: 1rem;
+    font-weight: 500;
+    line-height: 28px;
```
Comment (zoef-yt) [inline components/alert.css:12]: use rem

File: components/alert.css [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    color: var(--black-dark);
+    font-weight: 600;
+    border: solid var(--light-gray) 1px;
+    border-left: solid var(--success) 6px;
+}
+
+.dismiss {
+    cursor: pointer;
+    flex-shrink: 0;
+    margin: 0 1rem 0 1rem;
```
Comment (zoef-yt) [inline components/alert.css:124]: use "margin: 0 1rem" first value is top-bottom second value is right-left.

File: components/avtar.css [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    box-shadow      :0px 0px 1px 3px #9f091d;
+}
+.av-margin-online{
+    border          :solid 2px white;
+    box-shadow      :0px 0px 3px 4px var(--success);
+}
+.av-margin-offline{
+    border          : solid 2px white;
+    box-shadow      : 0px 0px 3px 4px var(--gray-dark);
+}
```
Comment (zoef-yt) [inline components/avtar.css:36]: maintain spacing consistency. i saw some place you have space after class block add here too

**Review by zoef-yt — CHANGES_REQUESTED** [review]

**Review by zoef-yt — APPROVED** [review]
Nice use of images in pr.  Good to merge after the changes.

**Review by developer-anand — APPROVED** [review]
Clean code. Good to merge.

**Pull request merged**